app/src/layout/index.tsx

```diff
@@ -13,14 +11,8 @@ export default function layout(
     element: <Layout />,
     id: "currentUser",
     async loader() {
-      try {
```

Comment:
This is unnecessary to special-case. Instead we use the general-purpose 403 handler in the ErrorPage.